cmd/broker/update_test.go

```diff
@@ -2401,3 +2401,128 @@ func TestMultipleUpdateNetworkFilterPersisted(t *testing.T) {
 	disabled = true
 	suite.AssertDisabledNetworkFilterRuntimeState(instance.RuntimeID, updateOperation2ID, &disabled)
 }
+
+func TestUpdateOnlyErsContext(t *testing.T) {
```

Comment:
small: rename the test like: TestUpdateOnlyErsContextForExpiredInstance or something like this (the "expiration" is mentioned in the name)